specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMaker.json

```diff
@@ -931,10 +1019,10 @@
       "description": "Post body schema for Replace KB operation.",
       "additionalProperties": false,
       "required": [
-        "qnAList"
+        "qnaList"
```

Comment:
fix this, as we got approval in v4.0

Reply:
same in below properties

Reply:
We fixed example to have "qnAList" not to break 4.0. But since, we are doing a breaking change, I think, we better fix it to be uniform across API calls. Kindly let me know.